THIRD-PARTY-NOTICES.txt

```diff
@@ -5,69 +5,1132 @@ reserves all other rights not expressly granted, whether by
 implication, estoppel or otherwise.
 
 In the event that we accidentally failed to list a required notice, please
-bring it to our attention. Post an issue or email us:
+bring it to our attention. Post an issue or email us: chakracore@microsoft.com
 
-           chakracore@microsoft.com
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do Not Translate or Localize
 
+ChakraCore incorporates third party material from the projects listed below.
+The original copyright notice and the license under which Microsoft received
+such third party material are set forth below. Microsoft reserves all other
+rights not expressly granted, whether by implication, estoppel or otherwise.
 
--------------------------------------------
+1.	backbone.suggestions (https://github.com/qloo/backbone.suggestions)
+2.	BeatDetektor.js
+3.	Catch
+4.	CRC.h
+5.	deltablue.js
+6.	Dojo Toolkit (https://dojotoolkit.org)
+7.	DSP.js (https://github.com/corbanbrook/dsp.js/)
+8.	ecma-proposal-object.getownpropertydescriptors (https://github.com/tc39/proposal-object-getownpropertydescriptors/)
+9.	File copyright Microsoft and Geoff Norton
+10.	File with code copyright Baron Schwartz
+11.	Files copyright .NET Foundation
+12.	Files with code copyright Paul Johnston (http://pajhome.org.uk/)
+13.	Intel VTune Amplifier XE (https://software.intel.com/en-us/intel-vtune-amplifier-xe)
+14.	JavaScript GameBoy Color Emulator
+15.	JetStream
+16.	jQuery (https://github.com/jquery/jquery)
+17.	Kraken (http://krakenbenchmark.mozilla.org)
+18.	MakeNewWorld() function
+19.	MochiKit (https://github.com/mochi/mochikit)
+20.	Mozilla XML-RPC Client code
+21.	Octane (https://github.com/chromium/octane)
+22.	pdfjs.js (https://github.com/mozilla/pdf.js)
+23.	Pixastic Lib (https://github.com/jseidelin/pixastic)
+24.	Prototype (https://github.com/sstephenson/prototype)
+25.	Sputnik  (https://github.com/fholm/IronJS/tree/master/Src/Tests/Sputnik/sputnik-v1)
+26.	SunSpider
+27.	test262 (https://github.com/ljharb/test262/)
```

Comment:
While we're at it, should we update the test262 URL to the new home at https://github.com/tc39/test262 ?